lib/run.js

```diff
@@ -94,14 +98,14 @@ async function run() {
                 }
                 await updateCheck(id, 'in-progress', {
                     title: checkName,
-                    summary: `linting took ${stdout.summary.duration}ms, errors: ${stdout.summary.error}, warnings: ${stdout.summary.warning}, info: ${stdout.summary.info}`,
+                    summary: summarize(stdout.summary),
                     annotations
                 });
             }
-            console.log(`clj-kondo detects some problems and exited with a non-zero code (${exitCode}). Please check founded problems at https://github.com/${GITHUB_REPOSITORY}/runs/${id}`);
```

Comment:
There is a lighter touch change here of something like:

> clj-kondo detected some problems and exited with a non-zero code (n). Please check the problems found at https://github.com/...

I shortened things because I'm a sucker for something succinct.